app/views/request/details.html.erb

```diff
@@ -41,7 +41,11 @@
     <tr class="<%= cycle('odd', 'even') %>">
     <% for column in columns %>
       <td data-label="<%= column %>"><%
-        d = info_request_event.send(column)
+        if column == 'details'
+          d = info_request_event.params_diff[:new].keys().join(', ')
```

Comment:
@gbp I don't _think_ there should be a problem with this since it's only the keys, but could you construct a query to pull out all possible values that might be displayed here? Just want any keys in the params column that _don't_ start with `old_` I think.

Reply:
```sql
=> SELECT DISTINCT key FROM (SELECT jsonb_object_keys(params) AS key FROM info_request_events) AS keys WHERE key NOT LIKE 'old_%';
             key
-----------------------------
 email
 title
 destination_request
 api
 visible
 reason
 token
 smtp_message_id
 subject
 prominence
 attachment_id
 prominence_reason
 comment
 user
 shared
 comment_if
 raw_data
 attachment
 deleted_incoming_message_id
 comments_allowed
 incoming_message
 handle_rejected_responses
 script
 tag_string
 actions
 embargo
 embargo_extension
 awaiting_description
 request
 explanation
 attention_requested
 message
 id
 described_state
 allow_new_responses_from
 outgoing_message
 user_url_name
 body
 stop_new_responses
 public_body_url_name
 questions
 deleted_outgoing_message_id
 event_created_at
 editor
```

Reply:
Yep sweet, nothing concerning there.